### ISSUE TITLE
Documentation in current folder is broken

### ISSUE THREAD

**Issue by thibaud-sanchez** [spring-cloud/spring-cloud-dataflow#4581]
**Description:**
If I follow the current link https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#api-guide-resources-server-meta-retrieving I have the current error into examples : 

```
Unresolved directive in api-guide.adoc - include::/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-docs/../spring-cloud-dataflow-classic-docs/target/generated-snippets/about-documentation/get-meta-information/http-request.adoc[]
```
There is no problem with this link : 
https://docs.spring.io/spring-cloud-dataflow/docs/2.7.x/reference/htmlsingle/#api-guide-resources-server-meta-retrieving

**Release versions:**
current

**Comment by sabbyanandan** [comment]
@thibaud-sanchez: Thanks for reporting. We will have a look.

**Comment by sabbyanandan** [comment]
@thibaud-sanchez: @jvalkeal just fixed the `current` docs. Let us know if you see any other issues. Thanks again!

**Comment by jvalkeal** [comment]
While this was fixed manually, I still need to do an actual fix for release workflows.

**Comment by jvalkeal** [comment]
This was fixed in #4586
